src/lang/en-US.json

```diff
@@ -751,8 +752,10 @@
   "time_s": "a second",
   "time_ss": "%d seconds",
   "time_m": "a minute",
+  "time_ms": "minute",
```

Comment:
just curious, what does the "s" mean here?

Reply:
I'm very open to changing the key here. I noticed we already had values for "a second", "a minute", etc until time_x, so I was sticking the singular word until time_xs, for "singular". 

Now that I look at it, that's not very clear. Any suggestions for a better key here? I'd like to avoid dropping the 'a' from the existing time_x keys to avoid going through and reworking wherever they're used, if possible.

Reply:
you should be able to use time_h or time_hh depending on the number of hours (no need to add a new key?)